src/main/java/com/faforever/client/fa/ProcessUtils.java

```diff
@@ -7,22 +7,14 @@
 import com.sun.jna.platform.win32.WinDef.DWORD;
 import com.sun.jna.platform.win32.WinNT;
 import com.sun.jna.platform.win32.WinNT.HANDLE;
-import lombok.experimental.UtilityClass;
 import lombok.extern.slf4j.Slf4j;
-import org.bridj.Platform;
 
 import java.lang.reflect.Field;
 
-@UtilityClass
 @Slf4j
 public class ProcessUtils {
 
-  void setProcessPriority(Process process, WindowsPriority priority) {
-    if (!Platform.isWindows()) {
-      log.debug("Setting process priority is only supported on Windows");
-      return;
-    }
```

Comment:
But the current code is still Windows only?

Reply:
Yes, I wrapped where it is called in a check for windows.

I didn't want to add injection to this class just for an os check.

Slightly not ideal since it isn't as low as it could be. So could be convinced.

Reply:
I suggest to put the Windows-exclusiveness into the method name then.